test/unit/specs/components/staking/PageDelegate.spec.js

```diff
@@ -1,45 +1,53 @@
-import setup from '../../../helpers/vuex-setup'
+import Vuex from 'vuex'
+import { mount, createLocalVue } from '@vue/test-utils'
 import PageDelegate from 'renderer/components/staking/PageDelegate'
 
+const localVue = createLocalVue()
+localVue.use(Vuex)
+
+const $route = {
+  params: {
+    delegate: '1a2b3c'
+  }
+}
+
 describe('PageDelegate', () => {
-  let wrapper, store, router
-  let {mount} = setup()
+  let wrapper
+  let store = new Vuex.Store({
```

Comment:
why did you choose to remove the helper?

Reply:
assuming i understand correctly... because the helper installs `localVue.use(VueRouter)` — `$route` becomes read only and cannot be set manually in the test.

https://vue-test-utils.vuejs.org/en/guides/using-with-vue-router.html

Reply:
but before it worked with `router.push('/staking/delegates/pubkeyX')` or not?

Reply:
See: https://github.com/cosmos/cosmos-ui/pull/449/files#diff-b0c8aefb908fde6e011139a1c7f71df6L27

Reply:
i think it worked before with `router.push` because there were no params involved.

Reply:
The route is defined as `/staking/delegates/:delegate` `:delegate` is the param you want to get in the component. When you push the route `/staking/delegates/pubkeyX` it will set the param `delegate` to `pubkeyX`. This is how it worked before.
As already mentioned, I would like to change the test setup back to using vuex-setup for consistency and to also test vuex-actions existence (lite e2e testing).

Reply:
OH! ok! let me try that out. that would be better!